### PR TITLE
feat: Add research template

### PR DESCRIPTION
**Pull request by ellsclytn**
For Infrastructure work at the very least, we often encounter the need
to work out how we want to go forward with something, but working it out
is quite a task in and of itself. So, like most things, we want to be
able to track the research work along with our other tasks.

---

## How I am structuring my commits

I am going to **write individual commits for review**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/research.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: 🍎 Research
```
Comment (jfhinchcliffe) [inline .github/ISSUE_TEMPLATE/research.md:2]: Apple, in English, commonly means an apple. I would _strongly_ advise against using this emoji, and instead use the research emoji.

Comment (jfhinchcliffe) [inline .github/ISSUE_TEMPLATE/research.md:2]:
```suggestion
name: 🎓 Research
```

Comment (ellsclytn) [reply .github/ISSUE_TEMPLATE/research.md:2]: Unfortunately no such emoji exists, though some alternatives could be:

:microscope:
:scientist: 
:nerd_face: 

And potentially others.

**Review by jfhinchcliffe — APPROVED** [review]
tentatively approved

File: .github/ISSUE_TEMPLATE/research.md (hunk 1 above)
Comment (jfhinchcliffe) [reply .github/ISSUE_TEMPLATE/research.md:2]: Let's make a ticket for this...

Comment (ellsclytn) [reply .github/ISSUE_TEMPLATE/research.md:2]: The mortarboard is used in graduations, which are a celebration of academic success. I don't think this is appropriate, as this implies the research __must__ be successful, which is not the case.

Comment (ellsclytn) [reply .github/ISSUE_TEMPLATE/research.md:2]: A research ticket?

**Pull request merged**